src/flint/fmpq.jl

```diff
@@ -390,14 +340,31 @@ end
 #
 ###############################################################################
 
+function cmp(a::QQFieldElemOrPtr, b::QQFieldElemOrPtr)
+  @ccall libflint.fmpq_cmp(a::Ref{QQFieldElem}, b::Ref{QQFieldElem})::Cint
+end
+
+function cmp(a::QQFieldElemOrPtr, b::ZZRingElemOrPtr)
+  @ccall libflint.fmpq_cmp_fmpz(a::Ref{QQFieldElem}, b::Ref{QQFieldElem})::Cint
```

Comment:
```suggestion
  @ccall libflint.fmpq_cmp_fmpz(a::Ref{QQFieldElem}, b::Ref{ZZRingElemElem})::Cint
```
At least I expect this from the signature

Reply:
Indeed, that's a bad copy&paste mistake :-(